cmd/create/cluster/cmd.go

```diff
@@ -2795,3 +2796,35 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 func getRolePrefix(clusterName string) string {
 	return fmt.Sprintf("%s-%s", clusterName, helper.RandomLabel(4))
 }
+
+func calculateReplicas(
+	isMinReplicasSet bool,
+	isMaxReplicasSet bool,
+	minReplicas int,
+	maxReplicas int,
+	privateSubnetsCount int,
+	isHostedCP bool,
+	multiAZ bool) (int, int) {
+
+	newMinReplicas := minReplicas
+	newMaxReplicas := maxReplicas
+
+	if !isMinReplicasSet {
+		if isHostedCP {
+			if privateSubnetsCount == 1 {
+				newMinReplicas = 2
+			} else {
+				newMinReplicas = privateSubnetsCount
+			}
+		} else {
+			if multiAZ {
+				newMinReplicas = 3
```

Comment:
I think you forgot to use the const MinReplicaMultiAZ

Reply:
+1